src/lib/services/notehub.ts

```diff
@@ -83,17 +83,17 @@ export async function updateDeviceEnvironmentVariablesByPin(
 
 // get events from Airnote project in Notehub
 export async function getEvents(deviceUID: string) {
-  /* this function is originally fetched on mount with 8 days' worth of data to
-		  populate the AQI average history with the previous week's data AND
+  /* this function is fetched on mount with 90 days' worth of data to
+		  populate the CSV download, the AQI average history AND
 		  display today's most current reading as well */
-  const TIMEFRAME = '8 days';
+  const TIMEFRAME = '90 days';
```

Comment:
I don’t love the idea of loading 90 days worth of data for every page load by default. It looks like it’s slowing down the page load by a considerable amount.

Comparing https://deploy-preview-61--airnote-live.netlify.app/dev:864475046544754/dashboard (an unrelated PR for comparison) and https://deploy-preview-63--airnote-live.netlify.app/dev:864475046544754/dashboard, the initial payload for this PR is ~400kb larger, and takes ~1–3 seconds longer to render on my high-end Macbook.

I think we should either make the page smart enough to load only the amount of data it needs—which probably involves decoupling the download so it does its own thing to get 90 days—or just set `TIMEFRAME` to 30 days (which would mean less download days, but would also make the performance difference far less noticeable).

Reply:
Done. I'll log a separate ticket to expand the CSV data downloads in the future